build_sdk.py

```diff
@@ -85,7 +85,20 @@ class ConfigInfo:
     #     examples = {
     #         "passive_server": Path("example/imx8mm/passive_server")
     #     }
-    # )
+    # ),
+    BoardInfo(
+        name="maaxboard",
+        gcc_cpu="cortex-a53",
+        loader_link_address=0x40480000,
+        kernel_options = {
+            "KernelPlatform": "imx8mq-evk",
```

Comment:
Shouldn't this be `maaxboard`? I assume that you've put it as `imx8mq-evk` as the version of seL4 that seL4CP uses doesn't have `maaxboard`. The solution is to update the the seL4 version used by seL4CP then.

However, the `maaxboard` works with `"KernelPlatform": "imx8mq-evk"`, why does it even exist as a separate platform in seL4?

Reply:
Basically there is the Core, the SoC and the Board (and then also the board variant ....) that you can target. If board use the same SoC, the binaries might work on the other boards also more or less But it's good practice to really select the most specific target at the moment. As far as I know, having generic images that are configured at runtime is not a goal the moment for the kernel. Not sure how the micokit plans to handle this, though.